pkg/api/validation/validation.go

```diff
@@ -2045,3 +2052,21 @@ func ValidateLoadBalancerStatus(status *api.LoadBalancerStatus) errs.ValidationE
 	}
 	return allErrs
 }
+
+// ValidateEvent makes sure that the event makes sense.
+func ValidateEvent(event *api.Event) errs.ValidationErrorList {
+	allErrs := errs.ValidationErrorList{}
+
+	requiresNamespace := true
+	if event.InvolvedObject.Kind == "Node" {
```

Comment:
This is kind of broken. What if we make events about a namespace, or other non-namespaced resource? Or change the name of Node?